### ISSUE TITLE
An error occurs during update of NVDA "next" snapshots

### ISSUE THREAD

**Issue by k-kolev1985** [nvaccess/nvda#8089]
### Steps to reproduce:
1. Install one of the "next" snapshots before the merge of Python 4. I'm not sure if it has necessarily to be of them, but my cases were such.
2. Try to update to the newest "next" snapshot, either via the update mechanism or via a downloaded install package from the snapshots page - it does not matter.

### Expected behavior:
The installation of the update should proceed as normal.

### Actual behavior:
At the end of the installation, NVDA brings a dialog stating that the update has failed and for details to look in the log file.

### System configuration:
- Operating system: Windows 10 Home version 1709 (build 16299.251), 32-bit, in Bulgarian with all locale settings set to "Bulgarian".
- NVDA version: next-14952,dd1cbe50.
- Machine brand and model: ASUS T100TA.
- Processor: Intel Atom CPU Z3740 at 1.33GHz.
- RAM Memory: 2.00GB.

### Additional information:
1. I'll attach the error log here. It says something about the system not being able to find some file.
2. Even though NVDA states in its error message that the update has failed, it seams that NVDA has been successfully updated, according to its "About" dialog.
3. I get this error not only on my ASUS laptop, but on two virtual machines as well. I don't get the error however on my desktop machine. The only difference between the machines on which it happens and the one on which it does not is the fact that the one on which it does not has 64-bit Windows and on the ones it does have 32-bit Windows installed. Though I don't know if that matters in this case or not.
[NVDA Error.txt](https://github.com/nvaccess/nvda/files/1814499/NVDA.Error.txt)

**Comment by Brian1Gaff** [comment]
Yes this is still occurring even after manually installing the previous next 
and then allowing it to  check and get the new one.

Sorry if its supposed to have been fixed. I'm going to do a manual install 
from
%temp%
Now.
 Brian

**Comment by k-kolev1985** [comment]
With the last "next" snapshot, now there is no error during the update process. So, the incorrect path to the registry was the issue.
